### ISSUE TITLE
Could not sync price

### ISSUE THREAD

**Issue by vncoelho** [CityOfZion/neo-scan#385]
API:
```js
2018-11-21 15:56:16.016 [warn] [neoscan_api@127.0.0.1] could not sync {"GAS", "USD", "3m"} price
2018-11-21 16:03:01.461 [warn] [neoscan_api@127.0.0.1] could not sync price
2018-11-21 16:04:31.558 [warn] [neoscan_api@127.0.0.1] could not sync price
2018-11-21 16:04:32.821 [warn] [neoscan_api@127.0.0.1] could not sync {"GAS", "BTC", "1d"} price
2018-11-21 16:05:01.460 [warn] [neoscan_api@127.0.0.1] could not sync price
2018-11-21 16:05:54.881 [warn] [neoscan_api@127.0.0.1] could not sync {"GAS", "USD", "1m"} price
2018-11-21 16:06:05.733 [warn] [neoscan_api@127.0.0.1] could not sync {"NEO", "USD", "1m"} price
2018-11-21 16:10:31.471 [warn] [neoscan_api@127.0.0.1] could not sync price
2018-11-21 16:14:07.351 [warn] [neoscan_api@127.0.0.1] could not sync {"NEO", "USD", "1d"} price
2018-11-21 16:14:33.276 [warn] [neoscan_api@127.0.0.1] could not sync {"NEO", "USD", "1d"} price
2018-11-21 16:22:31.562 [warn] [neoscan_api@127.0.0.1] could not sync price
2018-11-21 16:28:50.365 [warn] [neoscan_api@127.0.0.1] could not sync {"NEO", "BTC", "1w"} price
2018-11-21 16:29:32.909 [warn] [neoscan_api@127.0.0.1] could not sync {"GAS", "USD", "3m"} price
2018-11-21 16:35:22.335 [warn] [neoscan_api@127.0.0.1] could not sync {"GAS", "BTC", "1d"} price
2018-11-21 16:38:31.593 [warn] [neoscan_api@127.0.0.1] could not sync price
2018-11-21 16:38:37.693 [warn] [neoscan_api@127.0.0.1] could not sync {"NEO", "USD", "1d"} price
2018-11-21 16:42:41.289 [warn] [neoscan_api@127.0.0.1] could not sync {"NEO", "BTC", "1m"} price
2018-11-21 16:42:47.371 [warn] [neoscan_api@127.0.0.1] could not sync {"NEO", "BTC", "3m"} price
2018-11-21 16:42:53.356 [error] [neoscan_api@127.0.0.1] Error in process #PID<0.9848.30> on node :"neoscan_api@127.0.0.1" with exit value:
{%Poison.SyntaxError{message: "Unexpected token at position 0: <", pos: nil, token: "<"}, [{Poison.Parser, :parse!, 2, [file: 'lib/poison/parser.ex', line: 57]}, {Poison, :decode!, 2, [file: 'lib/poison.ex', line: 83]}, {HTTPoison.Base, :response, 7, [file: 'lib/httpoison/base.ex', line: 478]}, {CryptoCompare.Util.ApiMini, :get_body, 2, [file: 'lib/crypto_compare/util/api_mini.ex', line: 35]}, {NeoscanCache.Cache, :sync_price_history, 3, [file: 'lib/neoscan_cache/cache.ex', line: 186]}, {NeoscanCache.Cache, :"-sync_price/0-fun-0-", 4, [file: 'lib/neoscan_cache/cache.ex', line: 157]}, {Enum, :"-reduce/3-lists^foldl/2-0-", 3, [file: 'lib/enum.ex', line: 1925]}, {NeoscanCache.Cache, :"-sync_price/0-fun-3-", 0, [file: 'lib/neoscan_cache/cache.ex', line: 153]}]}

2018-11-21 16:43:01.648 [warn] [neoscan_api@127.0.0.1] could not sync price
2018-11-21 16:43:07.749 [warn] [neoscan_api@127.0.0.1] could not sync {"NEO", "USD", "1d"} price
2018-11-21 16:43:10.646 [error] [neoscan_api@127.0.0.1] Error in process #PID<0.11681.30> on node :"neoscan_api@127.0.0.1" with exit value:
{%Poison.SyntaxError{message: "Unexpected token at position 0: <", pos: nil, token: "<"}, [{Poison.Parser, :parse!, 2, [file: 'lib/poison/parser.ex', line: 57]}, {Poison, :decode!, 2, [file: 'lib/poison.ex', line: 83]}, {HTTPoison.Base, :response, 7, [file: 'lib/httpoison/base.ex', line: 478]}, {CryptoCompare.Util.ApiMini, :get_body, 2, [file: 'lib/crypto_compare/util/api_mini.ex', line: 35]}, {NeoscanCache.Cache, :sync_price_history, 3, [file: 'lib/neoscan_cache/cache.ex', line: 186]}, {NeoscanCache.Cache, :"-sync_price/0-fun-0-", 4, [file: 'lib/neoscan_cache/cache.ex', line: 157]}, {Enum, :"-reduce/3-lists^foldl/2-0-", 3, [file: 'lib/enum.ex', line: 1925]}, {NeoscanCache.Cache, :"-sync_price/0-fun-3-", 0, [file: 'lib/neoscan_cache/cache.ex', line: 153]}]}

2018-11-21 16:43:27.934 [error] [neoscan_api@127.0.0.1] GenServer NeoscanCache.Cache terminating
** (Poison.SyntaxError) Unexpected token at position 0: <
    (poison) lib/poison/parser.ex:57: Poison.Parser.parse!/2
    (poison) lib/poison.ex:83: Poison.decode!/2
    (httpoison) lib/httpoison/base.ex:478: HTTPoison.Base.response/7
    (crypto_compare) lib/crypto_compare/util/api_mini.ex:35: CryptoCompare.Util.ApiMini.get_body/2
    (neoscan_cache) lib/neoscan_cache/cache.ex:163: NeoscanCache.Cache.sync_price_details/0
    (neoscan_cache) lib/neoscan_cache/cache.ex:150: NeoscanCache.Cache.sync_price/0
    (neoscan_cache) lib/neoscan_cache/cache.ex:113: NeoscanCache.Cache.handle_info/2
    (stdlib) gen_server.erl:637: :gen_server.try_dispatch/4
Last message: :sync_price
2018-11-21 16:43:28.123 [info] [neoscan_api@127.0.0.1] Application neoscan_cache exited: shutdown
{"Kernel pid terminated",application_controller,"{application_terminated,neoscan_cache,shutdown}"}
Kernel pid terminated (application_controller) ({application_terminated,neoscan_cache,shutdown})

Crash dump is being written to: erl_crash.dump...done
```

sync:
```js
2018-11-21 17:20:43.877 [info] [neoscan_sync@127.0.0.1] 0.4 blocks/s, 0.4 transactions/s, 4524.5 download avg time, 4593.0 insert avg time
2018-11-21 17:20:43.877 [info] [neoscan_sync@127.0.0.1] 0 ratio_insert, 0 ratio_download
2018-11-21 17:20:48.878 [info] [neoscan_sync@127.0.0.1] 0.4 blocks/s, 0.4 transactions/s, 33649.0 download avg time, 9850.5 insert avg time
2018-11-21 17:20:48.878 [info] [neoscan_sync@127.0.0.1] 0 ratio_insert, 0 ratio_download
```

postgres:
```js
PostgreSQL init process complete; ready for start up.

2018-11-21 12:08:24.329 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
2018-11-21 12:08:24.329 UTC [1] LOG:  listening on IPv6 address "::", port 5432
2018-11-21 12:08:24.334 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
2018-11-21 12:08:24.375 UTC [58] LOG:  database system was shut down at 2018-11-21 12:08:24 UTC
2018-11-21 12:08:24.388 UTC [1] LOG:  database system is ready to accept connections
```

It looks like that only the api crashed.

**Comment by some-dev** [comment]
i'v got similar proplem. any recipe?

**Comment by fabwa** [comment]
SHould be fixed in #392
